ci/install-package-dependencies.sh

```diff
@@ -145,6 +124,23 @@ sudo chmod 777 /tools/verible
 tar -C /tools/verible -xf "$verible_tar" --strip-components=1
 export PATH=/tools/verible/bin:$PATH
 
+# Install verilator
+if [ $lsb_sr = "18.04" ]; then
+  UBUNTU_SUFFIX="-u18"
+fi
+
+VERILATOR_TARBALL=verilator"$UBUNTU_SUFFIX-v$VERILATOR_VERSION".tar.gz
```

Comment:
As before, if we merge code that depends on a new version of verilator, we should ask others to rebase their PRs so they get the update in the version of azure-pipelines.yml that they use for their CI runs.

Reply:
We're grabbing the binary from a different source but this PR doesn't change the version of Verilator that we are using so I think it shouldn't affect anyone using OT. We'll do an announcement when we update to v4.228.